README.md

```diff
@@ -399,6 +399,17 @@ make terraform
 which will display a plan and ask you to confirm it. Please consult the 
 Terraform documentation for details.
 
+If an environment is using Elasticsearch resources that are not shared by other
+environments, and you are deploying this environment for the first time, you
+will need to run
+
+```
+make deploy
+make terraform
+```
+
+again.
```

Comment:
```suggestion
again. If you don't, the deployment will work but it will run slower and incur extra cost. Running the `deploy` and `terraform` targets twice is only necessary when the deployment is first created.
```